korman/exporter/physics.py

```diff
@@ -77,6 +77,14 @@ def generate_physical(self, bo, so, bounds, name=None):
             physical.sceneNode = self._mgr.get_scene_node(bl=bo)
 
             getattr(self, "_export_{}".format(bounds))(bo, physical)
+            
+            if self._exporter().has_coordiface(bo):
+                if not physical.mass:
+                    # ...Since the object has a coordinate interface but isn't a kickable, it's likely going to be animated. In such case,
+                    # we must warn Plasma to have the collisions follow the object's coordinates as well.
+                    physical.mass = 1.
+                    simIface.setProperty(plSimulationInterface.kPinned, True)
+                    physical.setProperty(plSimulationInterface.kPinned, True)
```

Comment:
I think I would handle this in the animation modifier. During the post_export step, it is guaranteed that all plasma objects are created and linked. So, you can check the coordinate interface for the child objects and set the masses on them. I would advise checking all of the detector code and ensuring that the pinned flag is set on all regions. MOUL doesn't need that flag because all detectors are kinematic in MOUL, meaning no forces are applied to the physical.